egs/aishell2/s5/local/nnet3/tuning/run_lstm_2a.sh

```diff
@@ -0,0 +1,170 @@
+#!/bin/bash
+
+#    This is the standard "lstm" system, built in nnet3; this script
```

Comment:
Was it necessary to include the non-chain nnet3 recipes?   They are generally quite a bit worse than the chain recipes. 
Some of the comments seem out of date (referring to cleanup).
Also: normally these directories would have results at the top, showing what the results are and how they differ from other results in the same experimental directory.
And it's a good idea to make a 'compare_wer.sh' script to compare the WER results and diagnostics in a neat way.  E.g. look at the mini_librispeech one.  You should always put the results in comments at the top of these types of files.